tests/runtime_tests.cpp

```diff
@@ -217,4 +221,26 @@ TEST_F(LambdaRuntimeTest, binary_response)
     EXPECT_EQ(expected_length, invoke_outcome.GetResult().GetPayload().tellp());
     delete_function(funcname);
 }
+
+TEST_F(LambdaRuntimeTest, crash)
+{
+    Aws::String const funcname = build_resource_name("crash_backtrace");
+    create_function(funcname, "crash_backtrace" /*handler_name*/);
+    Model::InvokeRequest invoke_request;
+    invoke_request.SetFunctionName(funcname);
+    invoke_request.SetInvocationType(Model::InvocationType::RequestResponse);
+    invoke_request.SetLogType(Model::LogType::Tail);
+
+    Model::InvokeOutcome invoke_outcome = m_lambda_client.Invoke(invoke_request);
+    EXPECT_TRUE(invoke_outcome.IsSuccess());
+    EXPECT_EQ(200, invoke_outcome.GetResult().GetStatusCode());
+    EXPECT_STREQ("Unhandled", invoke_outcome.GetResult().GetFunctionError().c_str());
+    Aws::Utils::Base64::Base64 base64;
+    auto decoded = base64.Decode(invoke_outcome.GetResult().GetLogResult());
+    std::string tail_logs(reinterpret_cast<char const*>(decoded.GetUnderlyingData()), decoded.GetLength());
+    std::cerr << tail_logs << std::endl;
+    EXPECT_TRUE(tail_logs.find("Stack trace (most recent call last):") != std::string::npos);
```

Comment:
nit: `EXPECT_NE` instead of `EXPECT_TRUE`

Reply:
will fix